static/src/stylesheets/inline/article-photo-essay.scss

```diff
@@ -0,0 +1,156 @@
+@import '../_vars';
+@import '../_mixins';
+
+.content--article {
+    a.u-underline.in-body-link--immersive {
+      color: $news-main-1;
+
+      &:hover {
+        border-bottom: 1px solid $news-main-2;
+      }
+    }
+
+    blockquote.photo-essay-block-quote {
+      font-family: $f-serif-headline;
+      font-style: normal;
+      margin: 0 0 $gs-baseline * 2;
+
+      &:not(.quoted) {
+        background-color: #f6f6f6;
```

Comment:
Could you choose a gray, which is already defined here http://guardian.github.io/pasteup-palette/demo/? We're trying not to add new colors, since the palette is already rich :)